### PR TITLE
Meta item extend

### PR DESCRIPTION
**Pull request by NikolaBorislavovHristov**

### CODE REVIEW

File: src/state_types/models/notifications.rs [hunk 1]
```diff
@@ -103,10 +103,12 @@ impl<Env: Environment + 'static> UpdateWithCtx<Ctx<Env>> for Notifications {
                                     // we have to get `now` somehow (environment or through a msg)
                                     // Alternatively, just set that when we add lib items
                                     .retain(|v| {
-                                        lib_item
-                                            .state
-                                            .last_vid_released
-                                            .map_or(false, |lvr| v.released > lvr)
+                                        v.released.map_or(false, |vr| {
```
Comment (Ivshti) [inline src/state_types/models/notifications.rs]: 1) are you sure this is the same logic
2) released should be mandatory; if someone didn't provide it, then we'll default to 0 (1970)
3) this shows the benefit of rust - changing one small thing in the codebase will typecheck every usage; if this was JS or even TS we'd be fucked

File: src/types/meta_item.rs [hunk 2]
```diff
@@ -54,8 +54,18 @@ pub struct MetaPreview {
     #[serde(default)]
     pub name: String,
     pub poster: Option<String>,
+    pub logo: Option<String>,
+    pub description: Option<String>,
+    pub release_info: Option<String>,
+    pub runtime: Option<String>,
+    pub released: Option<DateTime<Utc>>,
```
Comment (Ivshti) [inline src/types/meta_item.rs]: I'm not sure `released` and `release_info` should be there together
have you checked whether this is consistent with the SDK documentation?

also, please comment in the code that some fields are obsolete and should be removed, like `genres` and `directors`

File: src/types/meta_item.rs [hunk 3]
```diff
@@ -100,13 +120,15 @@ pub struct Video {
     pub id: String,
     #[serde(alias = "name")]
     pub title: String,
-    pub released: DateTime<Utc>,
+    pub released: Option<DateTime<Utc>>,
```
Comment (Ivshti) [inline src/types/meta_item.rs]: I'm very much against this change, I said why earlier

File: src/types/meta_item.rs (hunk 2 above)
Comment (NikolaBorislavovHristov) [reply src/types/meta_item.rs]: released is the date it was released.
release info is string like `2007-2009` or `2007-`

Comment (NikolaBorislavovHristov) [reply src/types/meta_item.rs]: genres and directors marked as deprecated

**Pull request merged**